visualization/src/main/kotlin/org/jetbrains/kotlinx/dl/visualization/letsplot/PlotHelpers.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 JetBrains s.r.o. and Kotlin Deep Learning project contributors. All Rights Reserved.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlinx.dl.visualization.letsplot
+
+import jetbrains.letsPlot.intern.Plot
+
+internal typealias TensorImageData = Array<Array<Array<FloatArray>>>
```

Comment:
Hmm, looks like we need common typealiases for this (don't mind, just my thoughts)